galaxy_ng/app/settings.py

```diff
@@ -112,3 +114,80 @@
 # Disable django guardian anonymous user
 # https://django-guardian.readthedocs.io/en/stable/configuration.html#anonymous-user-name
 ANONYMOUS_USER_NAME = None
+
+# temporary until env is set in https://github.com/pulp/pulp_installer/pull/719
+os.environ['GALAXY_ENABLE_API_ACCESS_LOG'] = True
+
+if os.environ['GALAXY_ENABLE_API_ACCESS_LOG']:
+    INSTALLED_APPS = ['automated_logging', 'dynaconf_merge']
+    MIDDLEWARE = ['automated_logging.middleware.AutomatedLoggingMiddleware', 'dynaconf_merge']
```

Comment:
you are overwritting `MIDDLEWARE` defined at line 3, I think you can just do 

```suggestion
    MIDDLEWARE.append("automated_logging.middleware.AutomatedLoggingMiddleware")
```